cms/jinja2/templates/components/streamfield/glossary_section_block.html

```diff
@@ -0,0 +1,27 @@
+{% from "components/accordion/_macro.njk" import onsAccordion %}
+
+<section id="{{ value.title|slugify() }}">
+
+    <h2 class="ons-u-fs-l">{{ value.title }}</h2>
+
+    {% set glossary_terms=[] %}
+
+    {% for glossary_term in value.content %}
+        {% do glossary_terms.append({
+        "title": glossary_term.title,
+        "content": glossary_term.definition|richtext
+        })
+        %}
```

Comment:
let's move this to the block's `get_context`

Reply:
[Move logic from template to get_context](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/5ff6012d0b58d89d6f5d3d56b6c34b63bfc6ff83)